enterprise/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Database models field validators.
+"""
+import os
```

Comment:
@asadiqbal08 I believe quality checker should yell at you to include `from __future__ import absolute_import, unicode_literals`.